index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-env browser */
+/* global _gaq, ga */
+import { on, find } from 'domassist';
+import aug from 'aug';
+
+class GATrack {
```

Comment:
on the flip side, should everything be static or should we create a `new GATrack(el)` so we don't have to pass element around